examples/compiler/01_lowering.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lowering\n",
+    "\n",
+    "In the first compilation pass, three main tasks are carried out: _indexification_, _substitution_, and _domain-alignment_.\n",
+    "\n",
+    "\n",
+    "### Indexification\n",
+    "\n",
+    "The _indexification_ consists of converting functions into actual arrays of accesses. An access always keeps a reference to its originating function. For instance, all accesses to `u` such as `u[t, x + 1]` and `u[t + 1, x − 2]`would store a pointer to the same, user-defined `Function` _u(t, x)_. This metadata is exploited throughout the various compilation passes. \n",
```

Comment:
"... Functions into Indexeds, that is actual array accesses. An Indexed ..."

Reply:
fixed